neonavigation_launch/launch/demo.launch

```diff
@@ -3,6 +3,13 @@
   <arg name="use_path_with_velocity" default="true" />
   <arg name="vel" default="0.5" />
 
+  <rosparam>
+    planner_3d:
+      remember_updates: true
+      hist_ignore_range: 0.0
+      hist_ignore_range_max: 1.5
+  </rosparam>
```

Comment:
I know this works fine, but shouldn't these parameters be located inside `<node pkg="planner_cspace" type="planner_3d" name="planner_3d">`?

`planner_3d` namespace is not constant.

Reply:
`navigation.launch` was originally a skeleton code for real usage, and `demo.launch` is for demonstration which should not be in `navigation.launch`.
However, currently `navigation.launch` is much specific for demonstration. It maybe better to directly put it into `neonavigation_launch/config/navigate.yaml`

Reply:
I'm going to move these parameters to `config/navigation.yaml` for now and reorganize `neonavigation_launch` in #398.